### PR TITLE
projects.html: Add popup view for projects

### LINKED ISSUE #4: Allow clicking cards to expand them

### PR DESCRIPTION
**Pull request by hemangsk**
Fixes #4

:tada:

### CODE REVIEW

**Comment by sils** [comment]
use full issue URL! coala.io/commits

File: partials/tabs/projects.html [hunk 1]
```diff
@@ -26,6 +26,60 @@
...
+
+  <div id="modal1" class="modal">
+    <div  class="modal-content-bear">
+        <div class="col-md-4 profile">
+            <div class="project-name">
+                {{currentProject.name}}
+            </div>
+            <div class="project-detail-element  reqs-proj">
+                <div ng-show="currentProject.requirements.length>0" class="small-heading">REQUIREMENTS</div>
+                <li class="candetect milestones" ng-repeat="candetect in currentProject.requirements">{{ candetect }}</li><br>
```
Comment (sils) [inline partials/tabs/projects.html]: candetect?

**Comment by sils** [comment]
Fixes? Closes :(

File: partials/tabs/projects.html [hunk 2]
```diff
@@ -26,6 +26,60 @@
...
+        </div>
+
+        <div class="col-md-8 dashboard">
+            <div class="project-description">
+                {{currentProject.desc}}
+            </div>
+            <div class="row milestones-row">
+            <div class="milestones medium-heading">MILESTONES</div>
+            <div class="small-heading">GSOC 2017 CODING STARTS</div>
+             <li class="milestone" ng-repeat="milestone in currentProject['milestones']['GSoC 2017 Coding Starts']">{{ milestone }}</li>
```
Comment (sils) [inline partials/tabs/projects.html]: uhm I'd like this to be dynamic for all possible milestones so we're not bound to GSoC with this system

File: partials/tabs/projects.html [hunk 3]
```diff
@@ -26,6 +26,60 @@
...
   </section>
+
+  <div id="modal1" class="modal">
+    <div  class="modal-content-bear">
+        <div class="col-md-4 profile">
+            <div class="project-name">
+                {{currentProject.name}}
+            </div>
+            <div class="project-detail-element  reqs-proj">
+                <div ng-show="currentProject.requirements.length>0" class="small-heading">REQUIREMENTS</div>
```
Comment (sils) [inline partials/tabs/projects.html]: use http://www.w3schools.com/cssref/pr_text_text-transform.asp

File: partials/tabs/projects.html (hunk 2 above)
Comment (hemangsk) [inline partials/tabs/projects.html]: This is resolved in latest commit

File: partials/tabs/projects.html [hunk 4]
```diff
@@ -26,6 +26,56 @@
...
     </div>
   </section>
+
+  <div id="modal1" class="modal">
+    <div  class="modal-content-bear">
+      <div class="col-md-4 profile">
+        <div class="project-name">
+          {{currentProject.name}}
+        </div>
+        <div class="project-detail-element  reqs-proj">
```
Comment (sils) [inline partials/tabs/projects.html]: you should use the ng-show on the outer divs,not on the inner elements IMO

Comment (hemangsk) [inline partials/tabs/projects.html]: Addressed in 6af9bdc

**Comment by sils** [comment]
6af9bdc is ready

**Pull request merged**